src/UserManager.ts

```diff
@@ -563,50 +563,30 @@ export class UserManager {
         return signoutResponse;
     }
 
-    public async revokeAccessToken(): Promise<void> {
+    public async revokeTokens(types = this.settings.revokeTokenTypes): Promise<void> {
```

Comment:
in order to keep consumer libraries simple, lets add here a type, like the other public functions have: `RevokeTokensArgs`